.github/workflows/lint.yml

```diff
@@ -270,12 +270,15 @@ jobs:
     runs-on: ubuntu-18.04
     container:
       # ubuntu18.04-cuda10.2-py3.6-tidy11
-      image: ghcr.io/pytorch/cilint-clang-tidy:e2cfc57ce4fa3a257a4b78fdfdc2b065c167b9c5
+      image: ghcr.io/pytorch/cilint-clang-tidy:63613951b506a55b21167261fc4bf0bcd78e40c2
     steps:
       - name: Checkout PyTorch
         uses: actions/checkout@v2
         with:
           fetch-depth: 0 # to allow tools/clang_tidy.py to do its thing
+      - name: Install dependencies
+        run: |
+          sudo -H pip3 install setuptools
```

Comment:
to shorten the lint time, the dependencies better baked into the docker image. the lint time will be critical soon that ci gating is rolling out.

Reply:
that made sense for installing clang-tidy in the first place since that took a few minutes, here I think it's nice to keep the PyTorch-specific stuff separate since `pip install` only takes a few seconds: e.g. https://github.com/pytorch/pytorch/runs/2824635588

Reply:
pytorch needs a few more deps than setuptools, I included a few in the dockerfile already, more like to keep it consistent and performant instead of spreading in two repos, e.g. https://github.com/pytorch/test-infra/blob/master/ci-docker-base/Dockerfile.cilint-clang-tidy#L29

Reply:
That line still exists in the docker image. I added this line because previously the lint run failed with `setuptools cannot be found`.